src/kOS/Binding/MissionSettings.cs

```diff
@@ -10,15 +10,15 @@ namespace kOS.Binding
     [Binding("ksp")]
     public class MissionSettings : Binding
     {
-        private VesselTarget ship;
         private SharedObjects sharedObj;
+        private VesselTarget ship => VesselTarget.CreateOrGetExisting(sharedObj);
 
         public override void AddTo(SharedObjects shared)
         {
             sharedObj = shared;
 
             shared.BindingMgr.AddGetter("CORE", () => new Core((kOSProcessor)shared.Processor, shared));
-            shared.BindingMgr.AddGetter("SHIP", () => ship ?? (ship = VesselTarget.CreateOrGetExisting(shared)));
```

Comment:
This was never counted (no ship.LinkCount++ after `??`) but probably never even executed, thanks to `Update` (ship assigned there).